src/dashboard/Alerts/actions.js

```diff
@@ -308,3 +417,20 @@ export function showSeleteAlertShape(showShapes) {
     dispatch(setShowSelectedGeojson(showShapes));
   };
 }
+
+/**
+ *Thunk function that  saves drawn geometry to state
+ *
+ * @function
+ * @name saveDrawnGeometryOperation
+ *
+ * @param {object} geometry
+ *
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+export function saveDrawnGeometryOperation(geometry) {
```

Comment:
Sorry for reviewing this but I think this thunk as issues. Why is it a thunk while you can just dispatch your action directly using dispatch in your components?

Reply:
Yes, I see. let me dispatch it inside my component